src/renderer/widgets/RangeInput.tsx

```diff
@@ -65,6 +65,16 @@ const RangeInput = ({
     }
   };
 
+  const onWheel = (e: React.WheelEvent) => {
```

Comment:
Can we add `WheelEvent` to the `import { ..., WheelEvent } from react` so we can reference it here simply as `WheelEvent`?